recipes/erlang/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "18.3" %}
+
+package:
+  name: erlang
+  version: {{ version }}
+
+source:
+  fn: otp_src_{{ version }}.tar.gz
+  url: http://erlang.org/download/otp_src_{{ version }}.tar.gz
+  sha256: "fdab8129a1cb935db09f1832e3a7d511a4aeb2b9bb3602ca6a7ccb9730d5c9c3"
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - m4
+    - perl
+    - readline 6.2*
+    - openssl 1.0.2g
+    - ncurses
+    - zlib 1.2*
+  run:
+    - readline 6.2*
+    - openssl 1.0.2g
+    - ncurses
+    - zlib 1.2*
+
+test:
+  commands:
+    - erl -version
+
+about:
+  home: http://www.erlang.org/
+  license: Erlang Public License 1.1 (http://www.erlang.org/EPLICENSE)
+  summary: "A programming language used to build massively scalable soft real-time systems with requirements on high availability."
+
+extra:
+  recipe-maintainers:
+    - scopatz
+    - jakirkham
```

Comment:
So, I don't really use Erlang. Though I am happy to see it added. I'll probably play second fiddle here.